packages/taiko-client/docs/swagger.json

```diff
@@ -0,0 +1,310 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Taiko Preconfirmation Server API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "name": "API Support",
+      "url": "https://community.taiko.xyz/",
+      "email": "info@taiko.xyz"
+    },
+    "license": {
+      "name": "MIT",
+      "url": "https://github.com/taikoxyz/taiko-mono/blob/main/LICENSE.md"
+    },
+    "version": "1.0"
+  },
+  "paths": {
+    "/healthz": {
+      "get": {
+        "consumes": ["application/json"],
+        "produces": ["application/json"],
+        "summary": "Get current server health status",
+        "operationId": "health-check",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/preconfBlocks": {
+      "post": {
+        "description": "Insert preconfirmation blocks by the given groups to the backend L2 execution engine, please note that\nthe AVS service should sort the groups and make sure all the groups are valid at first.",
```

Comment:
```suggestion
        "description": "Insert a group of transactions into a tentative block for preconfirmation. If the group is the first for a block, a new tentative block will be created. Otherwise, the transactions will be appended to the existing tentative block. The API will fail if: 1) the block is not tentative, 2) any transaction in the group is invalid or a duplicate, 3) block-level parameters are invalid or do not match the current tentative block’s parameters, 4) the group ID is not exactly 1 greater than the previous one, or 5) the last group of the block indicates no further transactions are allowed.",
```

Reply:
Updated